libpam_misc/pam_misc.pc.in

```diff
@@ -0,0 +1,11 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
```

Comment:
Since neither `prefix` nor `exec_prefix` is used in this file, there is no need to define them.